application/app/Http/Controllers/Admin/UsersController.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use App\Models\User;
+
+class UsersController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index(Request $request)
+    {
+        $query = User::query();
+        $name = $request->get('name', null);
+        $email = $request->get('email', null);
+        $sort = $request->get('sort', "id-asc");
+        $pageUnit = $request->get('page_unit', 10);
+
+        switch ($sort){
+            case 'id-asc':
+                $query = $query->orderBy('id', 'ASC');
+                break;
+            case 'id-desc':
+                $query = $query->orderBy('id', 'DESC');
+                break;
+            case 'name-asc':
+                $query = $query->orderBy('name', 'ASC');
+                break;
+            case 'name-desc':
+                $query = $query->orderBy('name', 'DESC');
+                break;
+            case 'email-asc':
+                $query = $query->orderBy('email', 'ASC');
+                break;
+            case 'email-desc':
+                $query = $query->orderBy('email', 'DESC');
+                break;
+        }
+
+        if($name != null){
+            $query->where('name','like','%'.$name.'%');
+        }
+        if($email != null) {
+            $query->where('email','like','%'.$email.'%');
+        }
+
+        if($pageUnit != null) {
+            $Users = $query->paginate($pageUnit);
```

Comment:
変数名のアッパーキャメルはやめてください。クラスと見間違えます。
キャメルケースにしてください。
```suggestion
            $users = $query->paginate($pageUnit);
```